ui/components/multichain/network-list-menu/add-rpc-url-modal/add-rpc-url-modal.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+  Box,
+  ButtonPrimary,
+  ButtonPrimarySize,
+  FormTextField,
+} from '../../../component-library';
+import {
+  BlockSize,
+  Display,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+const AddRpcUrlModal = () => {
+  const t = useI18nContext();
+
+  return (
+    <Box padding={4}>
+      <FormTextField
+        id="additional-rpc-url"
+        label={t('additionalRpcUrl')}
+        labelProps={{
+          children: undefined,
```

Comment:
do we need to specify the value of children to undefined here? doesn't this take undefined by default ?

Reply:
i got a type error without it

<img width="727" alt="Screenshot 2024-06-25 at 8 18 38 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/93cb2883-6df1-4402-8739-59a60152990b">